src/pl/plpython/plpy_main.c

```diff
@@ -80,6 +82,24 @@ cancel_pending_hook_type prev_cancel_pending_hook;
 void PLy_handle_cancel_interrupt(void);
 
 bool PLy_enter_python_intepreter = false;
+bool PLy_path_added = false;
+
+/* GUC variables */
+#if PY_MAJOR_VERSION >= 3
+char *plpython3_path = NULL;
+
+bool 
+plpython3_check_python_path(char **newval, void **extra, GucSource source) {
+	if (PLy_path_added)
```

Comment:
I still don't understand why cannot we use `inited`.

Even user sets it twice, as long as `PLy_initialize` has not been executed, it doesn't take effect yet.

Reply:
because I think the logic is user may not create `plpython3u` language or call the function.
they just load `plython3u` then the `inited` can not access